siad/daemon_test.go

```diff
@@ -52,4 +103,12 @@ func TestUnitProcessConfig(t *testing.T) {
 			t.Error("processing failure at check", i, 2)
 		}
 	}
+
+	// Test invalid configs.
+	invalidModule := "z"
+	config.Siad.Modules = invalidModule
+	_, err := processConfig(config)
+	if err == nil {
```

Comment:
I would prefer to see a specific error here. I know the std library has a function "os.IsErrNotExist()" or something similar, maybe we can manage customized errors that way. I think you would want to do it by making a specific type + interface, though it would be nice if there was a less cody way of accomplishing that.